NuGet.config

```diff
@@ -2,9 +2,6 @@
 <configuration>
   <packageSources>
     <clear />
-    <add key="AspNetCore" value="https://dotnet.myget.org/F/aspnetcore-ci-dev/api/v3/index.json" />
-    <add key="AspNetCoreTools" value="https://dotnet.myget.org/F/aspnetcore-tools/api/v3/index.json" />
-    <add key="roslyn" value="https://dotnet.myget.org/F/roslyn/api/v3/index.json" />
-    <add key="NuGet" value="https://api.nuget.org/v3/index.json" />
+    <!-- Restore sources should be defined in build/sources.props. -->
```

Comment:
Will `dotnet restore` inside of a project still use the NuGet feeds defined in build/sources.props?

Reply:
Yes.

Reply:
To clarify, the `RestoreSources` property is a feature of NuGet, not KoreBuild or any particular build system.